zebra/zebra_vty.c

```diff
@@ -3016,7 +3017,20 @@ DEFPY(show_evpn_neigh, show_neigh_cmd, "show ip neigh",
 
 {
 	zebra_neigh_show(vty);
+	return CMD_SUCCESS;
+}
+
+DEFPY (show_evpn_arp_redirect,
+       show_evpn_arp_nd_redirect_cmd,
+       "show evpn arp-nd-redirect[json]",
+       SHOW_STR
+       "EVPN\n"
+       "ARP/NA packet redirect\n"
```

Comment:
ARP/ND?

Reply:
this was intentional I believe.  So I'm not going to change it.